Running an indexer processor (#768)
aptos-labs/developer-docs: apps/nextra/pages/en/build/indexer/indexer-sdk/documentation/run-processor.mdx

---
title: "Running Your Processor"
---

# Running Your Processor

## Pre-requisites

> You've already followed the [Creating a Processor](./create-processor.mdx) guide and have a processor set up that connects your steps.
> If you haven't done that yet, please follow the previous guides before proceeding.

Reviewer: Maybe simplify it a bit for translation purposes.  Basically giving the user the action to first read the previous guide.

```suggestion
- Please first read [Creating a processor](./create-processor.mdx), which will set up your processor and connect your processor steps.
```

Author: we have translators 🤯

Reviewer: We don't have translators, but I try as a whole to keep the language simple and straightforward, so when someone translates it to understand it in a different language, it can be simple to handle.
